### ISSUE TITLE
autocomplete not working

### ISSUE THREAD

**Issue by sravyakusam** [microsoft/vscode#108963]
Issue Type: <b>Bug</b>

HI, I have :
 "editor.quickSuggestions": {
        "other": true,
        "comments": false,
        "strings": true
    },

Still I dont get autocomplete suggestions in strings. THis is in a json file. How can I fix the issue..? 

VS Code version: Code 1.50.1 (d2e414d9e4239a252d1ab117bd7067f125afd80a, 2020-10-13T15:06:15.712Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz (8 x 2112)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.92GB (1.89GB free)|
|Process Argv|--crash-reporter-id 797033c0-e112-4d56-a855-c62b422139d2|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
code-gnu-global|aus|0.2.2
shell|bbe|0.3.0
gitlens|eam|10.2.2
remote-wsl|ms-|0.50.1
azure-account|ms-|0.9.3
cpptools|ms-|1.0.1
powershell|ms-|2020.6.0
team|ms-|1.161.1
vim|vsc|1.17.1

</details>

**Comment by aeschli** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/32480

Quick suggestions only appear at the end of a word, but JSON has a special word rule that includes the `"`.
So if you remove the ending "'", it will work.

**Comment by sravyakusam** [comment]
Hi Aeshcli, I could remove " at the end of the line in json, and see that autocomplete suggestions show up. However I followed the thread #32480, but couldnot see any update if the feature is implemented, to enable intellisense suggestions in the middle of the word. 

This was one proposal I saw. 

_So, instead of the boolean configuration value we have today, ("editor.quickSuggestions": true) we could have an enum, ala endOfWord, insideWords, everywhere._


Is this implemented ? or something else that enables intellisense  suggestions in the middle of the word.
